decoration.sk

```diff
@@ -732,30 +739,39 @@ caves and cliffs update part 1:
 	glow item frame [item]¦s = minecraft:glow_item_frame[relatedEntity=glow item frame]
 	
 	(grass|dirt) path [block]¦s = minecraft:dirt_path
-
+	any dirt = any dirt, dirt path
 
 the wild update:
 	minecraft version = 1.19 or newer
 
 	frogspawn¦s = minecraft:frogspawn
-	
+
+	mangrove root¦s = minecraft:mangrove_roots
+	muddy mangrove roots = minecraft:muddy_mangrove_roots
+	hanging roots = minecraft:hanging_roots
```

Comment:
These already exist at https://github.com/SkriptLang/skript-aliases/blob/e99be898254965b0207992f66a3289ab6744106e/building.sk#L784

If you want to move them to this file, they should be put in a 1.17 category

Reply:
They were added in 1.19

Reply:
Oh sorry, it linked the wrong thing. I meant: https://github.com/SkriptLang/skript-aliases/blob/e99be898254965b0207992f66a3289ab6744106e/decoration.sk#L710

Reply:
This still needs resolved. This addon can just be removed since this alias is declared *earlier* in this file :)